aws_pricing/datadog_checks/aws_pricing/data/conf.yaml.example

```diff
@@ -0,0 +1,22 @@
+init_config:
+
+instances:
+  -
+
+    ## @param region_name - string - optional
```

Comment:
```suggestion
    ## @param region_name - string - optional - default: us-east-1
```

Reply:
we should add it as placeholder also then

Reply:
You're right 👍